aws-ssm-maintenancewindow/aws-ssm-maintenancewindow.json

```diff
@@ -75,10 +75,16 @@
             "additionalProperties": false,
             "properties": {
                 "Key": {
-                    "type": "string"
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 128,
+                    "pattern": "^([\\p{L}\\p{Z}\\p{N}_.:/=+\\-@]*)$"
```

Comment:
Would we be able to address the constraint that tag key cannot be prefixed with "aws:" in the pattern? Those tags will override our system tags.

Reply:
will address this. Thanks!